src/dipdup/config.py

```diff
@@ -557,6 +581,15 @@ def contract_configs(self) -> List[ContractConfig]:
                 raise RuntimeError('Config is not initialized')
         return cast(List[ContractConfig], self.contracts)
 
+    @property
+    def entrypoints(self) -> Set[str]:
+        entrypoints = set()
+        for handler in self.handlers:
+            for pattern in handler.pattern:
+                if isinstance(pattern, OperationHandlerTransactionPatternConfig) and pattern.entrypoint:
+                    entrypoints.add(pattern.entrypoint)
```

Comment:
This is not entirely correct, consider the following example:
```
index:
  contracts:
    - my_contract
  handlers:
    - pattern:
        - destination: my_contract
          entrypoint: tezToToken
        - destination: token_contract
          entrypoint: transfer
```
Expected ep set: `[ 'tezToToken' ]`
Actual: `[ 'tezToToken', 'transfer' ]`